node/db/src/memory.rs

```diff
@@ -8,11 +8,12 @@ use fvm_ipld_blockstore::Blockstore;
 use parking_lot::RwLock;
 use std::collections::{hash_map::DefaultHasher, HashMap};
 use std::hash::{Hash, Hasher};
+use std::sync::Arc;
 
 /// A thread-safe `HashMap` wrapper.
-#[derive(Debug, Default)]
+#[derive(Debug)]
 pub struct MemoryDB {
-    db: RwLock<HashMap<u64, Vec<u8>>>,
+    db: Arc<RwLock<HashMap<u64, Vec<u8>>>>,
 }
```

Comment:
I think both `Clone` and `Default` can be derived.

Reply:
Unfortunately, they can't

Reply:
Why not?

Reply:
Didn't build for some reason. Fixed and now it does